packages/core/src/controllers/events.ts

```diff
@@ -39,6 +39,32 @@ export class EventClient extends IEventClient {
     );
   }
 
+  public init: IEventClient["init"] = async () => {
+    if (typeof process !== "undefined" && process.env.IS_VITEST === "true") return;
```

Comment:
Why is this needed? 

Also I believe we have this exact check somewhere else already so if we need it in multiple places it's safer to abstract it into a util fn; avoids e.g. forgetting the `typeof` check and/or updating lots of inline checks if we ever need to change this.

Reply:
good idea to make it an util.
 >Why is this needed?

so we don't spam logs on dev/ci runs